tests/test_framerate_regulator.py

```diff
@@ -58,3 +63,15 @@ def test_sleep():
 
     assert regulator.called == 200
     assert after - before >= 2.0
+
+
```

Comment:
the 2 tests below indicate there's an issue with calling them directly, something that should be probably fixed (in this or other PR).